component/componenttest/obsreporttest.go

```diff
@@ -86,6 +86,12 @@ func (tts *TestTelemetry) CheckReceiverMetrics(protocol string, acceptedMetricPo
 	return checkReceiverMetrics(tts.reader, tts.id, protocol, acceptedMetricPoints, droppedMetricPoints)
 }
 
+// CheckScraperLogs checks that for the current exported values for logs scraper metrics match given values.
+// Note: SetupTelemetry must be called before this function.
+func (tts *TestTelemetry) CheckScraperLogs(receiver component.ID, scraper component.ID, scrapedLogRecords, erroredLogRecords int64) error {
```

Comment:
I am actually planning to remove the other one. So I don't think this is necessary.

Reply:
@bogdandrutu This might be a dumb question: Do you know if there are already some example test codes I can follow to assert metric values? Or should I copy the helper functions to my own test package?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/12086

Reply:
Thanks @bogdandrutu , updated in https://github.com/open-telemetry/opentelemetry-collector/pull/12036/commits/fc21ef2ff66531442987223c30621d6febbcb2b2